extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/DebeziumCdcIntegrationTest.java

```diff
@@ -83,26 +82,22 @@ public void mysql() throws Exception {
             pipeline.readFrom(source)
                     .withNativeTimestamps(0)
                     .<ChangeRecord>customTransform("filter_timestamps", filterTimestampsProcessorSupplier())
-                    .groupingKey(record -> (Integer) record.key().toMap().get("id"))
-                    .mapStateful(
-                            LongAccumulator::new,
-                            (accumulator, customerId, record) -> {
-                                long count = accumulator.get();
-                                accumulator.add(1);
-                                Operation operation = record.operation();
-                                RecordPart value = record.value();
-                                Customer customer = value.toObject(Customer.class);
-                                return entry(customerId + "/" + count, operation + ":" + customer);
-                            })
-                    .setLocalParallelism(1)
-                    .writeTo(Sinks.map("results"));
+                    .filter(record -> record.operation() != UNSPECIFIED)
+                    .map(record -> {
```

Comment:
Could you help me out understand what was the change here?

Reply:
Instead of reading in order, then applying grouping (which can destruct order) and saving the result with "ordered" keys (but still order can be destructed by grouping), we directly map incoming events to the list. Test is just way simpler with less tendency to be flaky